addons/account_check_printing/models/account_payment.py

```diff
@@ -88,7 +88,7 @@ def print_checks(self):
                 }
             }
         else:
-            self.filtered(lambda r: r.state == 'draft').post()
+            self.filtered(lambda r: r.state == 'draft').move_id.post()
```

Comment:
this is a fix that should be in a separated commit. Also, plz use `self.filtered(lambda r: r.state == 'draft').action_post()`

Reply:
Removed from here since I should do another PR as a bugfix